scripts/pom.xml

```diff
@@ -129,40 +129,24 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>integration-testing</id>
+            <phase>integration-test</phase>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <executable>bash</executable>
+          <commandlineArgs>${project.basedir}/src/test/bash/all-tests</commandlineArgs>
+          <workingDirectory>${project.build.directory}</workingDirectory>
+        </configuration>
+      </plugin>
     </plugins>
   </build>
-  <profiles>
-    <profile>
-      <id>integration-test</id>
-      <activation>
-        <os>
-          <family>!Windows</family>
-        </os>
-        <property>
-          <name>!skipTests</name>
-        </property>
```

Comment:
Please do not remove this feature.
When using maven it is a common standard that you can disable the tests using `-DskiptTests=true`.
With your change, you removed this feature.
Please keep the plugin execution in the profile and just remove`<os>...</os>`.

Also please test the build locally on windows (`mvn clean install`):
* Does it work and find `bash` automatically?
* Does it work if you have installed shellcheck locally on windows?

Reply:
I have added the `!skipTests` feature again. The build runs locally on windows and find `bash` automatically. The `shellcheck` also works. (I used chocolatey to install shellcheck)